CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # bedrock-kms-http ChangeLog
 
+## 2.4.0 - 2020-07-15
```

Comment:
Thanks for updating the changelog!  However, the way our publish script works is that the person running the script must make a change to the changelog, and in a case like this, we would want to *just* change the date.  So, when you do the changelog like this, please set the date to "TBD"